python/hsfs/feature_view.py

```diff
@@ -0,0 +1,608 @@
+                            following formats `%Y%m%d`, `%Y%m%d%H`, `%Y%m%d%H%M`, `%Y%m%d%H%M%S`,  or `%Y%m%d%H%M%S%f`.
+                    storage_connector: Storage connector defining the sink location for the
+                        training dataset, defaults to `None`, and materializes training dataset
+                        on HopsFS.
+                    location: Path to complement the sink storage connector with, e.g if the
+                        storage connector points to an S3 bucket, this path can be used to
+                        define a sub-directory inside the bucket to place the training dataset.
+                        Defaults to `""`, saving the training dataset at the root defined by the
+                        storage connector.
+                    version: Version of the training dataset to retrieve, defaults to `None` and
+                        will create the training dataset with incremented version from the last
+                        version in the feature store.
+                    description: A string describing the contents of the training dataset to
+                        improve discoverability for Data Scientists, defaults to empty string
+                        `""`.
+                    data_format: The data format used to save the training dataset,
+                        defaults to `"csv"`-format.
+                    coalesce: If true the training dataset data will be coalesced into
+                        a single partition before writing. The resulting training dataset
+                        will be a single file per split. Default False.
+                    splits: A dictionary defining training dataset splits to be created. Keys in
+                        the dictionary define the name of the split as `str`, values represent
+                        percentage of samples in the split as `float`. Currently, only random
+                        splits are supported. Defaults to empty dict`{}`, creating only a single
+                        training dataset without splits.
+                    seed: Optionally, define a seed to create the random splits with, in order
+                        to guarantee reproducability, defaults to `None`.
+                    statistics_config: A configuration object, or a dictionary with keys
+                        "`enabled`" to generally enable descriptive statistics computation for
+                        this feature group, `"correlations`" to turn on feature correlation
+                        computation and `"histograms"` to compute feature value frequencies. The
+                        values should be booleans indicating the setting. To fully turn off
+                        statistics computation pass `statistics_config=False`. Defaults to
+                        `None` and will compute only descriptive statistics.
+                    train_split: If `splits` is set, provide the name of the split that is going
```

Comment:
I think we could make "train" the default here.

Reply:
@davitbzh If there is no splits, can `train_split` be not `None`?

Reply:
Since currently we are setting it to `None`, I can change it to `train` later after @davitbzh answer.